release-schema.json

```
@@ -80,7 +71,7 @@
       "properties": {
         "scheme": {
           "title": "Scheme",
-          "description": "The list, register or system from which the identifier is taken.",
+          "description": "The list, register or system from which the identifier is taken. If the scope of the list, register or system from which the identifier is drawn is national, it is suggested to follow the pattern {​ISO 3166-1 alpha-2}-{​system}. If it is subnational, it is suggested to follow the pattern {​ISO 3166-2}-{​system}.",
```

```suggestion
          "description": "The list, register or system from which the identifier is taken. If the scope of the list, register or system is national, it is suggested to follow the pattern {​ISO 3166-1 alpha-2}-{​system}. If it is subnational, it is suggested to follow the pattern {​ISO 3166-2}-{​system}.",
```
Just feels like some unnecessary repetition